migrations/1614180464492-AddV4SignedOrdersAndPesistentSignedOrders.ts

```diff
@@ -0,0 +1,111 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class AddV4SignedOrdersAndPesistentSignedOrders1614180464492 implements MigrationInterface {
+    name = 'AddV4SignedOrdersAndPesistentSignedOrders1614180464492';
+
+    public async up(queryRunner: QueryRunner): Promise<void> {
```

Comment:
feeling suspicious of the autogenerated migration. Could you confirm it's tested on some kind of prod replica (either local or staging) and it works?

Reply:
It looks fine on staging currently, but we will verify carefully when testing tomorrow 👍